app/dashboard/templates/profiles/trust-vue.html

```diff
@@ -75,6 +75,13 @@ <h5 class="modal-title" id="exampleModalLabel">Generate your Proof of Personhood
           @modal-dismissed="hideModal()">
         </twitter-verify-modal>
 
+        <qd-modal
```

Comment:
@thelostone-mc im a vue noob. can u help me make a simple connector modal here that connects the user on http://gitcoin.co/trust to the QD mission page pls?

Reply:
@thelostone-mc can u help here?  i just want it to show whether a user has any QD trustbonus here (instead of `serviceDict.twitter.is_verified` it should be `serviceDict.QD.is_verified`) and point the user at http://localhost:8000/quadraticlands/mission/diplomacy to get more trust

Reply:
this is wired in now ! thanks @gdixon for lending a hand here